NOTICE.md

```diff
@@ -0,0 +1,13 @@
+© 2022 The MITRE Corporation.
+ 
+Approved for Public Release; Distribution Unlimited. Case Number 18-3678.
+ 
+NOTICE
+ 
+MITRE grants permission to reproduce, distribute, modify, and otherwise use this software to the extent permitted by the licensed terms provided in the LICENSE.md file included with this project.
+ 
+This software was produced by The MITRE Corporation for the U. S. Government under contract. As such the U.S. Government has certain use and data rights in this software. No use other than those granted to the U. S. Government, or to those acting on behalf of the U. S. Government, under these contract arrangements is authorized without the express written permission of The MITRE Corporation.
```

Comment:
I thought we still needed to write the contract numbers, but just have it be a link to a website or some other common location.